docs/compute-element/install-htcondor-ce.md

```diff
@@ -220,8 +220,8 @@ For information on how to troubleshoot your HTCondor-CE, please refer to the ups
 Registering the CE
 ------------------
 
-To contribute to the the OSG Production Grid, your CE must be
-[registered with the OSG](https://github.com/opensciencegrid/topology#topology).
+To contribute capacity to collaborations, your CE must be
```

Comment:
As above, maybe just drop “to collaborations”?

Reply:
Fixed